index.html

```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
         </ul>
       </div>
     </section>
+    <section id="portfolio">
+      <div class="title">
+        <h2>My Recent Works</h2>
+        <hr />
+      </div>
+      <div class="plchdr">
+        <div class="img">
+          <img src="/images/plc-img.jpg" alt="" />
+        </div>
+        <div class="text">
+          <h2>Multi-Post-Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul>
+            <li><a href="">css</a></li>
+            <li><a href="">html</a></li>
+            <li><a href="">bootstrap</a></li>
+            <li><a href="">ruby</a></li>
+          </ul>
+          <button>See Project</button>
+        </div>
+      </div>
+      <div class="card cd1">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li><a href="">html</a></li>
+          <li><a href="">bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+      <div class="card cd2">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li><a href="">html</a></li>
+          <li><a href="">bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+      <div class="card cd3">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li><a href="">html</a></li>
+          <li><a href="">bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+        <button>See Project</button>
+      </div>
+      <div class="card cd4">
+        <h3>Professional Art Printing Data</h3>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li><a href="">html</a></li>
+          <li><a href="">bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+        <button>See Project</button>
+      </div>
```

Comment:
- [ ] Please make sure your see project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 ![Screenshot from 2022-10-28 10-03-18](https://user-images.githubusercontent.com/92303891/198561604-0d202740-3ec3-48c3-bd51-4f73d6f92d63.png)